build/assets/css/main.css

```diff
@@ -2,105 +2,219 @@
 html {
   color: #222;
   font-size: 1em;
-  line-height: 1.4; }
-
+  line-height: 1.4
```

Comment:
There are plenty of instances here we could have made used of variables and mixings if this was compiled done using sass. I think making it sass would give us more control and ease of use for others